Makefile

```diff
@@ -195,9 +195,7 @@ minikube-install: gadget-default-container kubectl-gadget
 	./kubectl-gadget undeploy || true
 	./kubectl-gadget deploy --hook-mode=auto \
 		--liveness-probe=$(LIVENESS_PROBE) \
-		--image-pull-policy=Never | \
-		sed 's/initialDelaySeconds: [0-9]*/initialDelaySeconds: '$(LIVENESS_PROBE_INITIAL_DELAY_SECONDS)'/g' | \
-		kubectl apply -f -
+		--image-pull-policy=Never
```

Comment:
Since you removed `LIVENESS_PROBE_INITIAL_DELAY_SECONDS`, you can remove it line 175 too.